outputs/yandex.go

```diff
@@ -0,0 +1,99 @@
+package outputs
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"html/template"
+	"log"
+	"os"
+	"time"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+
+	"github.com/falcosecurity/falcosidekick/types"
+)
+
+// NewYandexClient returns a new output.Client for accessing the Yandex API.
+func NewYandexClient(config *types.Configuration, stats *types.Statistics, promStats *types.PromStatistics, statsdClient, dogstatsdClient *statsd.Client) (*Client, error) {
+
+	if config.Yandex.AccessKeyID != "" && config.Yandex.SecretAccessKey != "" {
+		t := template.New("fieldname example")
+		t, _ = t.Parse(`[yandex-account]
+; work profile
+aws_access_key_id = {{.AccessKeyID}}
+aws_secret_access_key = {{.SecretAccessKey}}
+            `)
+		type AwsKeyPair struct {
+			AccessKeyID     string
+			SecretAccessKey string
+		}
+		p := AwsKeyPair{AccessKeyID: config.Yandex.AccessKeyID,
+			SecretAccessKey: config.Yandex.SecretAccessKey,
+		}
+		homeDirName, err := os.UserHomeDir()
+		os.MkdirAll(homeDirName+"/.aws", os.ModePerm)
+		file, err := os.Create(homeDirName + "/.aws/credentials")
+		if err != nil {
+			log.Println("create file: ", err)
+
+		}
+		err = t.Execute(file, p)
+		if err != nil {
+			log.Print("execute: ", err)
+
+		}
+
+	}
```

Comment:
I don't understand the purpose of this, why are you creating an `.aws/config` file? Can't you use env vars like for AWS Client?
Please, do not write anything on disk, keep everything in memory.

If it's just for avoiding a conflict with AWS credentials aside, you can create use this function for setting `Credentials`: [credential.NewStaticCredentials](https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/#NewStaticCredentials)